lib/src/snapd_client.dart

```diff
@@ -1094,6 +1094,15 @@ class SnapdClient {
     return SnapdConnectionsResponse._fromJson(result);
   }
 
+  /// Refreshes the snaps given by [names].
+  /// Returns the change ID for this operation, use [getChange] to get the status of this operation.
+  Future<String> bulkRefresh(List<String> names) async {
```

Comment:
Looking at the snapd source the terminology they used here is "refreshMany" - it's probably a good idea to match that in case a new "bulk" concept is introduced in the future.

Reply:
Thanks :+1:!